provisioning/service/src/Models/EnrollmentGroup.cs

```diff
@@ -154,37 +89,27 @@ public EnrollmentGroup(string enrollmentGroupId, Attestation attestation)
         }
 
         /// <summary>
-        /// Enrollment Group Id.
+        /// Enrollment group Id.
         /// </summary>
         /// <remarks>
         /// A valid enrollmentGroup Id shall be alphanumeric, lowercase, and may contain hyphens. Max characters 128.
         /// </remarks>
         /// <exception cref="InvalidOperationException">If the provided string does not fit the enrollmentGroup Id requirements</exception>
-        [JsonProperty(PropertyName = "enrollmentGroupId")]
+        [JsonProperty("enrollmentGroupId")]
         public string EnrollmentGroupId { get; internal set; }
 
         /// <summary>
         /// Current registration state.
         /// </summary>
-        [JsonProperty(PropertyName = "registrationState", DefaultValueHandling = DefaultValueHandling.Ignore)]
+        [JsonProperty("registrationState", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public DeviceRegistrationState RegistrationState { get; internal set; }
 
         /// <summary>
-        /// Attestation Mechanism.
+        /// Attestation mechanism.
         /// </summary>
-        [JsonProperty(PropertyName = "attestation")]
+        [JsonProperty("attestation")]
         private AttestationMechanism _attestation;
 
-        /// <summary>
-        /// Convert this object in a pretty print format.
-        /// </summary>
-        /// <returns>The string with the content of this class in a pretty print format.</returns>
-        public override string ToString()
-        {
-            string jsonPrettyPrint = JsonConvert.SerializeObject(this, Formatting.Indented);
-            return jsonPrettyPrint;
-        }
-
         /// <summary>
         /// Getter and setter for Attestation.
```

Comment:
nit: should we use lowercase here?

Reply:
I mean, I think the property doc comment is terrible. Not part of this PR, though.